Sources/VLCEditController.swift

```diff
@@ -111,6 +119,16 @@ extension VLCEditController: VLCEditToolbarDelegate {
         })
     }
 
+    func editToolbarDidAddToPlaylist(_ editToolbar: VLCEditToolbar) {
+        guard mediaLibraryService.playlists().isEmpty else {
+            delegate?.editController(editController: self,
+                                     present: addToPlaylistViewController)
+            return
+        }
+
+        addToNewPlaylist()
```

Comment:
the guard is weird here. I would've expected a simple if else :)

Reply:
Yes, at first it was a simple if else but I'm not sure why I changed it. 😄